### DISCUSSION TITLE
Disabling battery, wifi volume flyout.

### DISCUSSION THREAD

**Discussion by seventhstorm** [valinet/ExplorerPatcher#234] [Q&A] [answered]
is there a way to disable the battery, wifi and volume flyout? Using startallback+this mod. Really sleek. Just finding the flyouts disturbing. We can use the control centre button (icon) to access the flyout menu

**Comment by valinet** [comment, marked as answer]
Try shell:::{05d7b0f4-2121-4eff-bf6b-ed3f69b894d9}\SystemIcons
